compiler/scenario-service/server/src/main/scala/com/digitalasset/daml/lf/ScenarioServiceMain.scala

```diff
@@ -117,6 +119,47 @@ object ScenarioService {
     Status.NOT_FOUND.withDescription(s" context $id not found!").asRuntimeException
 }
 
+sealed trait ScriptStream {
```

Comment:
```suggestion
sealed abstract class ScriptStream {
```
just following the principal, always use the least powerful feature.